Java_Training/src/main/java/PalindromeString.java

```diff
@@ -7,21 +7,32 @@ public static void main(String[] args) {
 		Scanner sc = new Scanner(System.in);
 		String str = sc.nextLine();
 		sc.close();
-		boolean is_palindrome = stringPalindrome(str, stringReverse(str));
-		System.out.println("String is palindrome "+ is_palindrome);
+		boolean isPalindrome = stringPalindrome(str, stringReverse(str));
+		System.out.println("String is palindrome "+ isPalindrome);
 	}
 
+	/**
+	 * This method will reverse the string
+	 * @param str
+	 * @return
+	 */
 	public static String stringReverse(String str) {
-		String tmp="";
+		StringBuffer tmp = new StringBuffer("");
```

Comment:
StringBuilder is enough here. StringBuffer has synchronized methods for thread safety. Using it here is additional overhead.